src/main/java/com/epam/test_generator/services/jenkins/JenkinsJobServiceImpl.java

```diff
@@ -43,16 +41,17 @@ public JenkinsJobServiceImpl(
             Map<String, Job> jobs = jenkinsServerFactory.getJenkinsServer().getJobs();
 
             List<CommonJenkinsJobResponse> response = jobs.values().stream()
-                .map(this::transform).collect(Collectors.toList());
+                .map(job -> transform(job)).collect(Collectors.toList());
             return response;
         } catch (IOException e) {
-            throw new RuntimeException(e);
+            e.printStackTrace();
         }
+        return null;
```

Comment:
usually it's better to return emptyList then null.

Kindly look at this method https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#emptyList()

You could find some additional information in the book of Robert Martin named Clean Code: A Handbook of Agile Software Craftsmanship (Chapter 7, section Don't return Null).
here the link https://books.google.ru/books?id=_i6bDeoCQzsC&pg=PA110&lpg=PA110&dq=martin+don%27t+return+null&source=bl&ots=ep4Kyl8aZ0&sig=tcCuFUMx9FjpOr7_2yWCRbUBzTE&hl=ru&sa=X&ved=0ahUKEwjblZvn1NLZAhVCGCwKHSyuDdUQ6AEIkgEwCQ#v=onepage&q=martin%20don't%20return%20null&f=false

Reply:
it's appear here from merge with develop, it's another task